param/cobalt.yaml

```diff
@@ -160,3 +160,8 @@ sensor:
             x: 0
             y: 0
             z: 0
+hydrophones:
+    positions:
+        x: 3
+        y: 3
+        z: 3
```

Comment:
If it is distance from origin then it's also not consistent with the simulator, which has the ref hydrophone at 0 0 0